beacon-chain/blockchain/process_block.go

```diff
@@ -656,24 +663,36 @@ func (s *Service) lateBlockTasks(ctx context.Context) {
 		return
 	}
 
-	s.headLock.RLock()
-	headBlock, err := s.headBlock()
-	if err != nil {
+	if headState.Version() >= version.EPBS {
```

Comment:
Don't you want to process `headState` up to the current slot? What if we've already passed the ePBS fork, but the head state is still on the Electra version

Reply:
If the head state is still in Electra then the headBlock will be fine to send the FCU arguments isn't it?